h/templates/activity/search.html.jinja2

```diff
@@ -35,10 +35,20 @@
 <div class="search-bucket-stats">
   {% if bucket.uri %}
     <div class="search-bucket-stats__key">
-        Address
+        URL
     </div>
-    <div class="search-bucket-stats__val search-bucket-stats__address">
-        {{ bucket.uri }}
+    <div class="search-bucket-stats__val search-bucket-stats__url">
+        <div><a class="search-bucket-stats__link"
+                href="{{ bucket.uri }}">{{ pretty_link(bucket.uri) }}</a></div>
```

Comment:
See `DocumentBucket` initialization. `bucket.uri` will either be an HTTP(S) URL or `None`, in which case this whole section is not rendered.